src/supported-locales.js

```diff
@@ -8,8 +8,10 @@ const locales = {
     'af': {name: 'Afrikaans'},
     'ar': {name: 'العربية'},
     'am': {name: 'አማርኛ'},
+    'an': {name: 'Aragonés'},
     'az': {name: 'Azeri'},
     'id': {name: 'Bahasa Indonesia'},
+    'bn': {name: 'বাংলা'},
     'be': {name: 'Беларуская'},
```

Comment:
In English, Bengali would be listed after Belarusian... but I guess Bānlā would be listed before Belaruskaya. Just wanted to check if this ordering was intentional.

Reply:
Yes we try to sort the languages according to how the native name sounds in English. So Japanese (sounds like Nihon), is sorted in the 'n's and Bangla is after Bahasa Indonesia, and before Belarusian. It's a bit arbitrary, so we just try to be close.